packages/admin/src/services/ApplicationService.ts

```diff
@@ -106,11 +106,26 @@ const handleSectionOrdering = async (
   );
 };
 
+const handleQuestionOrdering = async (
+  sessionId: string,
+  applicationId: string,
+  sectionId: string,
+  questionId: string,
+  increment: number
+) => {
```

Comment:
If you're passing this many arguments into a function it's generally easier to just have one arg that's an object with all these properties - makes it way easier to add and remove args, and prevents mistakes with ordering the arguments correctly.

Reply:
Agreed

Reply:
Realised I have no tests for the post behaviour so I'll add these too